pilot/pkg/security/authn/policy_applier.go

```diff
@@ -19,16 +19,14 @@ import (
 
 	"istio.io/api/security/v1beta1"
 	"istio.io/istio/pilot/pkg/model"
-	"istio.io/istio/pilot/pkg/networking"
+	"istio.io/istio/pilot/pkg/networking/plugin"
 )
 
 // PolicyApplier is the interface provides essential functionalities to help config Envoy (xDS) to enforce
 // authentication policy. Each version of authentication policy will implement this interface.
 type PolicyApplier interface {
-	// InboundFilterChain returns inbound filter chain(s) for the given endpoint (aka workload) port to
-	// enforce the underlying authentication policy.
```

Comment:
This line seems deleted accidentally?

Reply:
It is intentionally, we moved the method?